docs/source/model_design/concept_model/standard_results_formats/index.rst

```diff
@@ -0,0 +1,242 @@
+      * female
+      * both
+  * - :code:`year`
+    - string
+    - Simulated year
+    - * 2022 
+      * 2023
+      * etc.
+
+      If year is not included as a stratifying variable in the simulation, the 
+      year parameter should still be specified in vivarium results data frames 
+      with a value of :code:`{start_year}_through_{end_year}`.
+  * - :code:`{other_stratifying_variables}` (example: :code:`treatment`)
+    - string
+    - Some other variable by which to stratify values (in the same way as age, 
+      sex, year, etc.). Intervention coverage is a common example.
+    - Name and values of {other_stratifying_variables} to be defined in advance 
+      of implementation by researchers.
+
+      Example:
+
+      * covered
+      * uncovered
+  * - :code:`measure`
+    - string
+    - Description of what is being measured about the entity
+    - * ylds
+      * ylls
+      * deaths
+      * person_time
+      * transition_count
+      * continuous_exposure
+  * - :code:`entity_type`
+    - string
+    - Type of entity being measured
+    - * cause
+      * rei
+      * modelable_entity
+      * {other}
+  * - :code:`entity`
+    - string
+    - Name of entity being measured
+    - :code:`gbd_{entity_type}_name.replace(" ","_").lower()` if there is a 
+      corresponding GBD entity, otherwise agreed upon between research and 
+      engineering teams
+      
+      * measles
+      * child_stunting
+      * hemoglobin
+  * - :code:`sub_entity`
+    - string
+    - Name of entity state or category
+    - * susceptible/infected/recovered
+      * cat1/cat2/cat3/cat4
+      * first_moment/second_moment
+  * - :code:`value`
+    - float
+    - Measured value
+    - (-inf, inf)
+
+.. note::
+
+  As a reminder, the definitions of the first and second moments for a 
+  continuous exposure measure, :math:`Y`, for our purposes are defined below. 
+
+  .. math::
+
+    \text{First moment} = \sum_{i}Y_i \times \text{person time}_i
+
+    \text{Second moment} = \sum_{i}Y_i^2 \times \text{person time}_i
+
+  Using these measures, we can then calculate the population mean and standard 
+  deviation values for that continuous exposure by dividing the first and second 
+  moment (respectively) by the sum of person time within the population.
+
+Some examples
++++++++++++++
+
+.. list-table:: Examples of measure and entity field values
+  :header-rows: 1
+
+  * - Result
+    - Measure
+    - Entity type
+    - Entity
+    - Sub-entity
+  * - YLDs due to acute myocardial infarction
+    - ylds
+    - cause
+    - ischemic_heart_disease
+    - acute_myocardial_infarction
+  * - Deaths due to diarrheal diseases
+    - deaths
+    - cause
+    - diarrheal_diseases
+    - infected
+  * - Person time in the susceptible to measles state
+    - person_time
+    - cause
+    - measles
+    - susceptible
+  * - Transitions from measles to recovered from measles
+    - transition_count
+    - cause
+    - measles
+    - infected_to_recovered
```

Comment:
```suggestion
    - infected__to__recovered
```
I suggest we request that the _from_ and _to_ states of a transition be separated by a unique string such as `'__to__'`, so that the two states can be easily extracted using `.split()`, e.g., `states = sub_entity.split('__to__')`. This is not possible with the current Vivarium naming scheme, which has transitions with names like `'diarrheal_diseases_to_susceptible_to_diarrheal_diseases'`. Even if we replace these annoying names with something simpler like `'infected_to_susceptible'`, there's no guarantee that we'll never have a complicated cause model with many cause states (such as the LTBI model), some of which could potentially include the word "to." So my proposal is to use double underscores for the separator, and to never use double underscores in the name of a state.

Reply:
This is an example of the downside of having every observer output a file with the same columns -- otherwise, the transition count observer could have two separate columns for this. But I agree that double underscore is better than single.

Reply:
Well I guess we could just have separate columns for the states of a transition, though with this change it will be easy enough to do ourselves. In the CIFF SAM project, since nothing really looked how I wanted, I [preprocessed the data](https://github.com/ihmeuw/vivarium_research_ciff_sam/blob/main/model_validation/ciff_sam_results.py#L159) into the form I wanted, including putting the states of the transitions into separate columns. In the current Vivarium output, the entire transition description is stored in the "measure" column.

Reply:
I think the point of conforming the output to the same format as much as possible is to avoid having to specify lots of special cases to the engineers. Ali and I added the `sub_entity` column as a sort of catch-all that could handle several types of entities such as causes, risks, and transitions, and hopefully it will provide a structure that can also capture new types of outputs that we haven't thought of yet, without us having to think to carefully about what we want. Having a more standardized formate reduces the burden of communication between researchers and engineers, which is often a sticking point. And as long as we understand what we're getting in the output and it's well-structured, it should be easy for us to write preprocessing code to reformat the Vivarium output into a form more suitable for answering research questions.